NotifyMe/services/service.py

```diff
@@ -0,0 +1,32 @@
+from NotifyMe.models.notification import Notification
+from NotifyMe.models.notificationType import NotificationType
+from NotifyMe.models.subscription import Subscription
+from NotifyMe.models.subscriptionPlan import SubscriptionPlan
+from NotifyMe.models.user import User
+
+class UserService:
+  def get_user_data(self, request):
+    try:
+      users = User.objects.all()
+      return users
+    except Exception as e:
```

Comment:
handle other exceptiona aks chat gpt , Exception should be the last to catch , use custom exception